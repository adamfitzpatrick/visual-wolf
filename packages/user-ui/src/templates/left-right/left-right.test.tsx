import React from 'react';
import { render, screen } from '@testing-library/react';
import { LeftRightTemplate } from '.';

describe('LeftRightTemplate', () => {
  test('should render content as provided', () => {
    
    const props = {
      header: <div>header</div>,
      sidebar: <div>sidebar</div>,
      main: <div>main</div>
    }
    const { container } = render(<LeftRightTemplate { ...props }/>)
    expect(container).toMatchSnapshot();
  });
})
