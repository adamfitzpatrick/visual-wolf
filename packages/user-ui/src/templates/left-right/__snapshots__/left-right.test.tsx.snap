// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LeftRightTemplate should render content as provided 1`] = `
<div>
  <div
    class="left-right-template"
  >
    <header>
      <div>
        header
      </div>
    </header>
    <section>
      <div>
        sidebar
      </div>
    </section>
    <main>
      <div>
        main
      </div>
    </main>
  </div>
</div>
`;
